zstacklib/zstacklib/network/ovs/controller.py

```
@@ -0,0 +1,546 @@
+            logger.error(f'Set port {port_name} failed. {err}')
+            raise OvsPortError(port_name, msg=str(err))
+
+    def set_interface(self, if_name: str, *options: str) -> None:
+        """Set interface options."""
+        _validate_name(if_name, "interface name")
+        try:
+            cmd = CTL_BIN + f'set Interface {if_name} '
+            for opt in options:
+                cmd += f'{opt} '
+            shell.call(cmd)
+        except Exception as err:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)